pkg/hive/manager.go

```diff
@@ -22,6 +21,7 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/dynamichelper"
 	utillog "github.com/Azure/ARO-RP/pkg/util/log"
 	"github.com/Azure/ARO-RP/pkg/util/uuid"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
our imports in files are structured as follows: 
* system library imports
* third party imports 
* first party imports (github.com/Azure/ARO-RP)

So the bottom line here needs to move up above to the third party import line.

Reply:
Thank you, the error messaging around this, was a lot less helpful.